assets/js/edit.js

```diff
@@ -337,25 +441,105 @@ class Edit extends Component {
 						</PanelRow>
 					</PanelBody>
 				</InspectorControls>
-				<div className={className}>
+				<div className="wp-block-podcasting-podcast-outer">
 					{src ? (
-						<figure key="audio" className={className}>
-							<audio controls="controls" src={src} />
-							{((caption && caption.length) || !!isSelected) && (
-								<RichText
-									tagName="figcaption"
-									placeholder={__(
-										'Write caption…',
-										'simple-podcasting'
+						<>
+							<div className="wp-block-podcasting-podcast__container">
+								{showImage && displayArt && (
+									<div className="wp-block-podcasting-podcast__show-art">
+										<div className="wp-block-podcasting-podcast__image">
+											<img
+												src={showImage}
+												alt={showName}
+											/>
+										</div>
+									</div>
+								)}
+
+								<div className="wp-block-podcasting-podcast__details">
+
+									{displayEpisodeNumber && episodeNumber && (
```

Comment:
This causes either no/turning off the episode number to display to also hide the title. I think what you want here is a check for the episode title display.

The eposide number check should be part of line 463